shared_model/cryptography/ed25519_sha3_impl/bindings/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Soramitsu Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if (SWIG)
+
+add_definitions(-DSWIG)
+
+find_package(SWIG REQUIRED)
+INCLUDE(${SWIG_USE_FILE})
+
+find_package(PythonLibs REQUIRED)
+include_directories(${PYTHON_INCLUDE_PATH})
+
+find_package(JNI REQUIRED)
+include_directories(${JAVA_INCLUDE_PATH})
+include_directories(${JAVA_INCLUDE_PATH2})
```

Comment:
why 2?

Reply:
JAVA_INCLUDE_PATH is the include path to jni.h, JAVA_INCLUDE_PATH2 is the include path to jni_md.h. Both are needed here.